### ISSUE TITLE
Perianal anesthesia

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9848]
**Preferred term label:**
Perianal anesthesia

**Synonyms**
Saddle

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Spinal diseases such as spinal epidural abscess

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
Perianal anesthesia
HP:0034982
